phys2bids/tests/test_utils.py

```diff
@@ -0,0 +1,98 @@
+"""
+    Runs unit tests on utils.py
+"""
+
+import os
+from phys2bids import utils
+
+
+# Tests check_input_dir
+def test_check_input_dir():
+    test_path = '/home/mrmeeseeks/blue/'
+    path_corrected = utils.check_input_dir(test_path)
+    assert path_corrected == test_path[:-1]
+
+
+# Tests check_input_ext
+def test_check_input_ext():
+    test_filename = 'help'
+    ext = 'py'
+    out_filename = utils.check_input_ext(test_filename, ext)
+    assert str(out_filename) == f'{test_filename}.{ext}'
+
+
+# Tests check_input_type
+def test_check_input_type():
+    test_filename = 'Test_belt_pulse_samefreq.acq'
+    test_path = '/home/travis/build/physiopy/phys2bids/phys2bids/tests/data'
+    assert utils.check_input_type(test_filename, test_path)
+
+
+# Tests path_exists_or_make_it
+def test_path_exists_or_make_it():
+    test_folder = '/home/travis/build/physiopy/phys2bids/tests/foo'
+    utils.path_exists_or_make_it(test_folder)
+    os.path.isdir(test_folder)
+    os.rmdir(test_folder)  # Removes the created folder
```

Comment:
Can we add a test for a path that already exists? You should just be able to call the function again with the same input before you use `os.rmdir`!

Also, the way this is currently written precludes running the tests locally (for example, I won't be able to create the directory `'/home/travis/build` on my Mac laptop). To get around this, I would recommend using `pytest`'s [`tmpdir` fixture](http://doc.pytest.org/en/latest/tmpdir.html)!

Reply:
I didn't know about this. Thank you for the heads-up @rmarkello !